### PR TITLE
Adding routing to other pages and adding css to those pages

### PR DESCRIPTION
**Pull request by aaronkitelinger**

### CODE REVIEW

**Comment by aaronkitelinger** [comment]
The program is starting to look good! I think that the LaborAction dto is still a bit too big and could be shortened or put into two different classes or just shorten the class in general. I also don't understand why you got rid of your ILaborAction and LaborActionStub classes which are used for testing and doing business logic. I added a fragments folder just in case you want to use fragments in any way shape or form. I added CSS to the other pages and routed them to specific endpoints
Your website is really easy to understand and does compile. 

I really started to understand the concept of routing to views on controllers and understanding it a lot more.
I also learned how to create auto-completes very easily.
And spring JPA and what it's for.

    @GetMapping("/layout")
    public String layout(){
        return "layout";
    }

I wrote the introduction to mental health help and selected a few disorders to add to the disorders database.

**Comment by ccbrowndev** [comment]
It appears you did this code review before we pushed our Sprint 2 content to master, which happened on Wednesday night. I apologize for it being available late.

File: src/main/java/com/laborhelper/enterprise/service/ILaborAction.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package com.laborhelper.enterprise.service;
+
+public interface ILaborAction {
+}
```
Comment (discospiff) [inline src/main/java/com/laborhelper/enterprise/service/ILaborAction.java:4]: This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.

File: src/main/java/com/laborhelper/enterprise/service/LaborActionServiceStub.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package com.laborhelper.enterprise.service;
+
+public class LaborActionServiceStub {
+}
```
Comment (discospiff) [inline src/main/java/com/laborhelper/enterprise/service/LaborActionServiceStub.java:4]: This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.

File: src/main/resources/templates/layout.html [hunk 3]
```diff
@@ -2,10 +2,10 @@
 <html lang="en" xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>Labor Helper</title>
+    <!Adding css to other pages>
```
Comment (discospiff) [inline src/main/resources/templates/layout.html:6]: This comment isn't needed, as the purpose is obvious from context.  We don't put many comments in HTML.

**Comment by discospiff** [comment]
Some good ideas in this branch, as well as a few items that appear to be not fully complete.  If the team likes these changes, I recommend keeping the branch open, and completing the features, before merging.

**Pull request closed without merging**